dev/breeze/src/airflow_breeze/prod/prod_params.py

```diff
@@ -47,10 +47,10 @@ class ProdParams:
     install_docker_context_files: bool
     disable_pypi_when_building: bool
     disable_pip_cache: bool
-    upgrade_to_newer_dependencies: str
-    skip_installing_airflow_providers_from_sources: bool
-    cleanup_docker_context_files: bool
-    prepare_buildx_cache: bool
+    upgrade_to_newer_dependencies: str = "false"
+    skip_installing_airflow_providers_from_sources: bool = False
```

Comment:
@potiuk `skip_installing_airflow_providers_from_sources`, `cleanup_docker_context_files`, ` prepare_buildx_cache`. Do we have to keep these flags `False` by default? Because when user doesn't pass them it will be False and that's why i didn't keep it default.

Reply:
It was failing when I manually run it at least  `./Breeze2 build-prod-image` :)

Reply:
Oh i see. It's working fine for me. I have checked again and it's working. Could you check once again and let me know? if it throws error, i will fix it

Reply:
Ah no - it's already merged and fixed :)